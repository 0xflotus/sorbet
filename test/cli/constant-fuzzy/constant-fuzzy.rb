# typed: true

Yieldr.foo(Intege, Int)

module A; Constant = 1; end;

module A::B; Constant; end
