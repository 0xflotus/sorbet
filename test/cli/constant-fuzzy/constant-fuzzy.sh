#!/bin/bash

main/sorbet test/cli/constant-fuzzy/constant-fuzzy.rb 2>&1
