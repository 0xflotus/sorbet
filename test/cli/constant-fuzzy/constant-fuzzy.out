test/cli/constant-fuzzy/constant-fuzzy.rb:3: Unable to resolve constant `Yieldr` http://go/e/5002
     3 |Yieldr.foo(Intege, Int)
        ^^^^^^
    https://git.corp.stripe.com/stripe-internal/sorbet/tree/master/rbi/core/_self.rbi#L__LINE__: Did you mean: `::Enumerator::Yielder`?
     895 |class Enumerator::Yielder < Object
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
test/cli/constant-fuzzy/constant-fuzzy.rb:3: Unable to resolve constant `Intege` http://go/e/5002
     3 |Yieldr.foo(Intege, Int)
                   ^^^^^^
    https://git.corp.stripe.com/stripe-internal/sorbet/tree/master/rbi/core/integer.rbi#L__LINE__: Did you mean: `::Integer`?
     2 |class Integer < Numeric
        ^^^^^^^^^^^^^^^^^^^^^^^
test/cli/constant-fuzzy/constant-fuzzy.rb:3: Unable to resolve constant `Int` http://go/e/5002
     3 |Yieldr.foo(Intege, Int)
                           ^^^
test/cli/constant-fuzzy/constant-fuzzy.rb:7: Unable to resolve constant `Constant` http://go/e/5002
     7 |module A::B; Constant; end
                     ^^^^^^^^
    test/cli/constant-fuzzy/constant-fuzzy.rb:5: Did you mean: `::A#Constant`?
     5 |module A; Constant = 1; end;
                  ^^^^^^^^
